transcom/mymove
# INT-B-19552

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A941851&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

Removed full PPM from TOO queue if they are moving from a duty location that provides services counseling

### How to test

1. Access the customer app and make a full ppm
2. Complete customer and services counselor steps
3. PPM should **not** be in TOO queue
4. make a new ppm from 32228 mayport FL
5. complete customer and services counselor steps
6. PPM should be in the TOO queue
